Sources/SwiftGRPCNIO/GRPCWebCorsHandler.swift

```diff
@@ -0,0 +1,73 @@
+import NIO
+import NIOHTTP1
+
+/// Handler that manages the CORS protocol for requests incoming from the browser.
+class GRPCWebCorsHandler {
```

Comment:
Super-nit: Should we spell `CORS` all-uppercase? We've been doing that with e.g. `GRPC`, but not sure which way is idiomatic.

Reply:
done